grassroots-backend/src/contacts/Contacts.service.ts

```diff
@@ -5,43 +5,36 @@ import {
   CreateContactInDto,
   PaginatedContactOutDTO,
   PaginatedContactSearchInDTO,
-} from "../grassroots-shared/Contact.entity.dto";
-import { InjectRepository } from "@mikro-orm/nestjs";
-import { EntityRepository, FilterQuery } from "@mikro-orm/core";
+} from "../grassroots-shared/Contact.entity.dto.js";
+import { EntityManager, FilterQuery } from "@mikro-orm/core";
 import { LikeOrUndefined } from "../util/LikeOrUndefined";
 
 @Injectable()
 export class ContactsService {
-  constructor(
-    @InjectRepository(ContactEntityOutDTO)
-    private readonly contactsRepository: EntityRepository<ContactEntityOutDTO>,
-    // Will be needed for explicit flushing.
-    // private readonly em: EntityManager,
-  ) {}
+  constructor(private readonly entityManager: EntityManager) {}
```

Comment:
This change and the above `useFactory` provider change above seem to be related. Both seem to be negative tradeoffs of moving to vitest (unless I'm miss reading). We used to be able to use `contactsRepository` which had type information about the entity it was interacting with, and we used to be able to inject `ContactsService` (and I'm assuming future Services) directly in tests.

Is this something that's worth documenting why this tradeoff exists and what the benefits are?

Reply:
I see that in #79 you address the module part of this comment. LGTM and iterate after these two PRs are in.